interface/portraits/portraits_highelfundead_sprites.gfx

```diff
@@ -0,0 +1,260 @@
+# hair, mouth, chin, nose, eyes, cheek, ear, neck
+spriteTypes = {
+
+############## highelf ##########################
+
+	spriteType = {
+		name = "PORTRAIT_highelfgfx_child_male"
+		texturefile = "gfx\\characters\\highelf_child\\highelf_child_male.dds"
+		noOfFrames = 1
+		norefcount = yes
+	}
+	spriteType = {
+		name = "PORTRAIT_highelfgfx_child_female"
+		texturefile = "gfx\\characters\\highelf_child\\highelf_child_female.dds"
+		noOfFrames = 1
+		norefcount = yes
+	}
+	spriteType = {
+		name = "PORTRAIT_highelffelgfx_child_male"
+		texturefile = "gfx\\characters\\highelf_child\\bloodelf_child_male.dds"
+		noOfFrames = 1
+		norefcount = yes
+	}
+	spriteType = {
+		name = "PORTRAIT_highelffelgfx_child_female"
+		texturefile = "gfx\\characters\\highelf_child\\bloodelf_child_female.dds"
+		noOfFrames = 1
+		norefcount = yes
+	}
```

Comment:
Child portraits declared twice: here and in `portraits_highelf_sprites`. The same with other sprites here.
Also, please move the left sprites to `portraits_highelf_sprites`.